pkg/apis/policy/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -218,6 +219,13 @@ type Policy struct {
 	// ConfigMapRef defines the reference to a configMap with the policy definition.
 	// +optional
 	ConfigMapRef *ConfigMapReference `json:"configMapRef,omitempty"`
+	// FetchConfigFile controls whether ConfigFile will be fetched and made
+	// available for CIP level policy evaluation. Note that this only gets
+	// evaluated (and hence fetched) iff at least one authority matches.
+	// The ConfigFile will then be available in this format:
+	// https://github.com/opencontainers/image-spec/blob/main/config.md
+	// +optional
+	FetchConfigFile *bool `json:"fetchConfigFile,omitempty"`
```

Comment:
Could you run `make docs` ? So we get the new field in the documentation.

Reply:
Snap, totes forgot about that. We should probably add a step in GitHub actions to check for this.